multiwallet.go

```diff
@@ -652,3 +661,14 @@ func (mw *MultiWallet) ChangePrivatePassphraseForWallet(walletID int, oldPrivate
 
 	return nil
 }
+
+// checkForSeedClash checks to see if any wallet in the multi wallet has the same seed as the passed seed
+// returns an error if there is any
+func (mw *MultiWallet) checkForSeedClash(encryptedSeed []byte) error {
+	for _, wal := range mw.wallets {
+		if bytes.Equal(wal.EncryptedSeed, encryptedSeed) {
```

Comment:
`EncryptedSeed` cannot be used to compare because the encrypted value is different for every encryption, `EncryptedSeed` is cleared once a wallet is backed up and `EncryptedSeed` is only saved for new wallets.